packages/mdc-textfield/outline/mdc-text-field-outline.scss

```diff
@@ -63,3 +63,15 @@
   mdc-text-field-transition(opacity);
   fill: transparent;
 }
+
+// Never show the idle outline when the label is floating, otherwise the label
```

Comment:
Is it correct to be putting this within the outline subcomponent CSS when it references classes outside that component? Otherwise this idea LGTM.

Reply:
eh youre probably right. I moved it to mdc-text-field.scss